lib/jsonschema-tools.js

```diff
@@ -1,19 +1,20 @@
 'use strict';
 
-const _               = require('lodash');
-const yaml            = require('js-yaml');
-const path            = require('path');
-const semver          = require('semver');
-const readdirSync     = require('recursive-readdir-sync');
-const fse             = require('fs-extra');
-const pino            = require('pino');
-const util            = require('util');
-const exec            = util.promisify(require('child_process').exec);
-const RefParser       = require('json-schema-ref-parser');
-const mergeAllOf      = require('json-schema-merge-allof');
-const jsonSchemaFaker = require('json-schema-faker');
-const traverseSchema  = require('json-schema-traverse');
-const Promise         = require('bluebird');
+const _                     = require('lodash');
+const yaml                  = require('js-yaml');
+const path                  = require('path');
+const semver                = require('semver');
+const readdirSync           = require('recursive-readdir-sync');
+const fse                   = require('fs-extra');
+const pino                  = require('pino');
+const util                  = require('util');
+const exec                  = util.promisify(require('child_process').exec);
+const RefParser             = require('json-schema-ref-parser');
+const mergeAllOf            = require('json-schema-merge-allof');
+const jsonSchemaFaker       = require('json-schema-faker');
+const traverseSchema        = require('json-schema-traverse');
+const jsonStableStringify   = require('json-stable-stringify');
```

Comment:
Looks good :] But let me take the opportunity to rant on this style of alignment :D
It is uncool having to change and review all these lines, just because one of them got added.
I know this is a controversial topic, and don't want to start a conversation, but just call this out!

Reply:
Looks like GH can ignore whitespace to reduce noise like this:
![image](https://user-images.githubusercontent.com/819437/176104964-fdb141fa-d1e9-4b72-9fb1-18687c9a4d44.png)